dspace/modules/api/src/main/java/org/dspace/JournalUtils.java

```diff
@@ -721,14 +721,14 @@ private static Manuscript manuscriptFromCrossRefJSON(JsonNode jsonNode) throws R
                 int month = 12;
                 if (dateParts.has(1)) {
                     month = dateParts.get(1).asInt();
-                    if (month > calendar.get(Calendar.MONTH)) {
+                    if (year > calendar.get(Calendar.YEAR) && month > calendar.get(Calendar.MONTH)) {
```

Comment:
This logic doesn't make sense. I think it should be 

if (year > calendar.get(Calendar.YEAR) || (year ==  calendar.get(Calendar.YEAR) && month > calendar.get(Calendar.MONTH))) {